### PR TITLE
Decompile los-control-h

### PR DESCRIPTION
**Pull request by Brent-Hickey**
I don't know what i'm doing but this one seemed easy

### CODE REVIEW

**Comment by xTVaser** [comment]
add to whitelist

File: decompiler/config/jak2_ntsc_v1.jsonc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   // if you want to filter to only some object names.
   // it will make the decompiler much faster.
-  "allowed_objects": [],
+  "allowed_objects": ["los-control-h"],
```
Comment (ManDude) [inline decompiler/config/jak2_ntsc_v1.jsonc:10]: don't change this

File: goal_src/jak2/engine/collide/los-control-h.gc [hunk 2]
```diff
@@ -5,5 +5,44 @@
...
+  (format #t "~1Tdst-proc: ~D~%" (-> obj dst-proc))
+  (format #t "~1Thave-los: ~D~%" (-> obj have-los))
+  (format #t "~1Thave-no-los: ~D~%" (-> obj have-no-los))
+  (format #t "~1Tcheck-interval: ~D~%" (-> obj check-interval))
+  (format #t "~1Tlast-check-time: ~D~%" (-> obj last-check-time))
+  (format #t "~1Tlast-collide-result: #<collide-tri-result @ #x~X>~%" (-> obj last-collide-result))
+  (format #t "~1Tcollide-with: ~D~%" (-> obj collide-with))
+  (label cfg-4)
+  obj
+  )
```
Comment (ManDude) [inline goal_src/jak2/engine/collide/los-control-h.gc:48]: method is unnecessary

**Review by ManDude — CHANGES_REQUESTED** [review]

File: decompiler/config/jak2_ntsc_v1.jsonc [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
   // if you want to filter to only some object names.
   // it will make the decompiler much faster.
-  "allowed_objects": [],
+  "allowed_objects": [""],
```
Comment (ManDude) [inline decompiler/config/jak2_ntsc_v1.jsonc:10]: Just get rid of this string.

**Pull request merged**